ikasaneip/pom.xml

```diff
@@ -1368,6 +1376,18 @@
                 <version>${version.logback.core}</version>
```

Comment:
Seems like a wrong bversion reference

Reply:
Not something that has been changed as part of this PR. 
`<version.logback.core>1.2.10</version.logback.core>` is currently on the 3.3.x version branch. Are you saying this is incorrect?